reports/management/commands/import-historical.py

```diff
@@ -8,25 +8,22 @@
 
 class Command(BaseCommand):
     help = 'Import historical GMMP data from an XLSX file'
-    option_list = BaseCommand.option_list + (
-        make_option('--global',
-            action='store_true',
-            help='Coverage is global'),
-        make_option('--region REGION',
-            action='store',
-            dest='region',
-            help='Import historical data for a region. One of: %s' % ', '.join(sorted(REGION_COUNTRY_MAP.keys()))),
-        make_option('--country COUNTRY',
-            action='store',
-            dest='country',
-            help='Import historical data for a country. One of: %s' % ', '.join(sorted(c[0] for c in get_countries()))),
-    )
+    def add_arguments(self, parser):
+        parser.add_argument('filenames', nargs='+', help='Excel file(s) to import')
+        parser.add_argument('--global', action='store_true', help='Coverage is global')
+        parser.add_argument('--region REGION', action='store', dest='region', help='Import historical data for a region. One of: %s' % ', '.join(sorted(REGION_COUNTRY_MAP.keys())))
+        parser.add_argument('--country COUNTRY', action='store', dest='country', help='Import historical data for a country. One of: %s' % ', '.join(sorted(c[0] for c in get_countries())))
+
 
     def handle(self, *args, **options):
         historical = Historical()
         coverage = None
         region = None
         country = None
+        filenames = options['filenames']
+
+        if not filenames:
+            raise CommandError("Must specify --filename")
```

Comment:
1. Is this necessary? Doesn't `nargs='+'` ensure _one or more_ filenames is passed.
2. There shouldn't be a need to specify `--filename` anymore: `import-historical --global filename1 filename2` should be how it works. Correct?

Reply:
I missed that, but it's been fixed @kilemensi